tests/cli/test_shell.py

```diff
@@ -11,7 +11,7 @@ class ShellTest(unittest.TestCase):
     # In this case we just want to go through the code to see if it's okay
 
     def test_shell_execution_memory_storage(self):
-        shell = Shell(argv=['--memory-storage'])
+        shell = Shell(argv=['--memory-storage', '--', '--extra-arg'])
```

Comment:
Should we allow extra unknown args?

Reply:
Current `Shell` behavior allows any kind of extra argument, so that's what I reflected in tests. But read https://github.com/HathorNetwork/hathor-core/pull/636#discussion_r1229700949

Reply:
Got it. Thanks!